common/hedera-common-nativesupport/src/main/java/com/hedera/common/nativesupport/NativeLibrary.java

```diff
@@ -155,12 +168,20 @@ public String locationInJar() {
      */
     @NonNull
     public static String locationInJar(
-            @NonNull final String libraryName, @NonNull final Map<OperatingSystem, String> libExtensions) {
+            @NonNull final String libraryName,
+            @NonNull final Map<OperatingSystem, String> libPrefixes,
+            @NonNull final Map<OperatingSystem, String> libExtensions) {
         Objects.requireNonNull(libraryName, "name must not be null");
+        Objects.requireNonNull(libPrefixes, "libPrefixes must not be null");
         Objects.requireNonNull(libExtensions, "libExtensions must not be null");
         final OperatingSystem os = OperatingSystem.current();
         final Architecture arch = Architecture.current();
 
+        String libPrefix = libPrefixes.get(os);
+        if (libPrefix == null) {
+            libPrefix = "";
+        }
```

Comment:
MAybe a.getOrDefault() to avoid some lines of code.
```suggestion
        final String libPrefix = libPrefixes.getOrDefault(os,"");
```

Reply:
done :)